packages/accounts/plugindefs/index.ts

```diff
@@ -0,0 +1 @@
+export { MultiOwnerPluginGenConfig } from "./multi-owner/config.js";
```

Comment:
we can probably do better here by just reading all the `config.js` files found within this folder and exporting their contents. we might need to do default exports for the configs though

Reply:
hmm but will take break the vscode module navigation ? or you mean reading all config.ts files and export their contents by writing into a new file for exports in `out` or something

Reply:
yea the way I was thinking would break module navigation good call... we could make the plugin gen script overwrite this file though that's a good idea